### ISSUE TITLE
[Accessibility] Info icon present under the Action tab is not accessible through keyboard.

### ISSUE THREAD

**Issue by shahparth-lab** [Azure/LogicAppsUX#3258]
### Describe the Bug with repro steps

[Keyboard Navigation- ICM- Actions tab]

User Experience:
If the info icon presents under tabs will not be accessible through keyboard, then the user who depends upon the keyboard/Screen Reader for navigation will not get the tooltip information appears on the info icon.

Note: Login Credentials must not be provided within the bug.

Repro Steps:
Launch URL: https://portal.microsofticm.com/imp/v5/automation/workflows/create
Sign in with valid credential.
Navigate and select 'Automation' option in header section.
Navigate to various tile like Learn more, Start from blank, ICM etc. under Automation page.
Navigate and select 'Start from blank'.
Navigate all the controls on the 'Create workflow' page.
Navigate to 'Search' edit field.
Search valid/invalid connectors.
Select any connector such as 'Schedule'.
Navigate all the controls in the 'Actions' tab.
Verify the issue.

Actual Result:
The info icons present under the Action tab are not accessible through keyboard. 
Observation: when keyboard focus land on info buttons by clicking Enter/Space is not accessible and it is accessible with mouse.

Expected Result:
The info icon present under the Action tab should be accessible with a keyboard, so that the user can easily get information about the tooltip.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Browser

OS Version: Windows 11 22H2 (OS Build:22000.978)
Browser: Chrome Version 106.0.5249.62 (Official Build) (64-bit)

AB#25072788

**Comment by hartra344** [comment]
This is on the ICM portal which is still currently using V1 portal, this won't be fixed or supported. I'm working with them to move to new designer.
